### PR TITLE
Update the Spi libraries

### PR DESCRIPTION
**Pull request by metinavci**

### CODE REVIEW

File: RamenPos/RamenPos.csproj [hunk 1]
```diff
@@ -38,14 +38,21 @@
     <Reference Include="log4net, Version=2.0.8.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
```
Comment (RaymondK-AP) [inline RamenPos/RamenPos.csproj]: can we remove log4net?

File: RamenPos/packages.config [hunk 2]
```diff
@@ -3,7 +3,9 @@
   <package id="log4net" version="2.0.8" targetFramework="net45" />
```
Comment (RaymondK-AP) [inline RamenPos/packages.config]: can we remove log4net?

Comment (metinavci) [reply RamenPos/packages.config]: Fixed

File: RamenPos/RamenPos.csproj (hunk 1 above)
Comment (metinavci) [reply RamenPos/RamenPos.csproj]: Fixed

**Pull request merged**